data/korath/korath missions.txt

```diff
@@ -257,3 +257,337 @@ mission "Small Scale Delivery"
+				to display
+					"flagship crew" > 2
+			`	While your crewmate works on refreshing the atmosphere with outside air, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+				to display
+					"flagship crew" == 2
+			`	One by one, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
+				to display
+					"flagship crew" < 2
+
+			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
+
+			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+				to display
+					"flagship crew" > 1
+			`	As a precaution, you make a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+				to display
+					"flagship crew" <= 1
+			branch accept
+	on complete
+		conversation
+			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch. You count the coffins together and show them the boxes of possessions.`
+			`	The officer thanks you and hands you a data chip that vaguely reminds you of a license.`
+			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, you presume their family.`
+			`	You go back inside your ship rather than intrude on their grief. You slot the data chip into your systems but the information is in an alien language. You set it aside and work on a salvage plan.`
+			`	As you review your notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage the <EDC-ship>. The closest system where you can find that many human speakers is on Greenwater in Hai space.`
```

Comment:
```suggestion
			`A crowd pours out of the spaceport when you land. It takes some time for the workers to establish barricades and clear a path so the transports can get to your cargo hold. A team in uniform greets you formally when you open the hatch, and you count the coffins together with then, then show them the boxes of possessions. An officer thanks you and hands you a data chip that vaguely reminds you of a license.`
			`	The crowd has been somber, but when the first coffin leaves your ship, they erupt in a cacophony of barking sounds. Some Korath try to rush forward but are held back by others, presumably their family.`
			`	Rather than intrude on their grief, you return to your ship, slotting the data chip you received into your systems. Unfortunately, the information is in an alien language, so you set it aside and work on a plan to salvage the <EDC-ship>. Gathering some notes on the ship and the number of crew they were carrying, it's clear that you will need to carry at least 66 additional crew that you can communicate with to be able to salvage it. The closest system that comes to mind with many human speakers is on Greenwater, in Hai space.`
```
As with earlier, too much mechanical split-up. Editing the last line to be more impactful on coming up with the idea, rather than just trailing onto it casually.

I'm stopping my first review here - this should be barred if you haven't made contact with the Hai yet. Either the *whole* mission series should be barred based on that, or there should be an alternative to going to the nearest human planet in human space if you haven't met the Hai.

Reply:
It is not possible to check if Greenwater has been *mapped*. You can check if the system was entered, but that's slightly unfair since the player has no way to know if they've ever entered the system.

I could do a PR to add the capability of checking if a system has been mapped, but then we'd be waiting on another PR. I think not having mapped Greenwater is a small enough niche case that it can be ignored in this PR for now.

Reply:
Not only can you check if a system was entered, but you can check if a player has visited a planet.

Reply:
That isn't fair either. You just need to know the planet exists and is Hai. For that, the minimum is that it must be mapped. If you found it by purchasing a map, then you won't have ever entered it.

Reply:
Oh, we do have it.

`"visited planet: <planet>"`

Reply:
I'm not sure what you're suggesting by "mapping" then.

Reply:
Actually, it turns out there are no outfitters that can map Greenwater, so this niche case possible anyway. It is sufficient to have `visited system: Fah Soom`. I have added that requirement.

Also, I was wrong about what HasSeen does. It just checks if you've seen the system, regardless of whether you know the name or contents.

Reply:
> I'm not sure what you're suggesting by "mapping" then.

You can put a system on your map two different ways:

1. Flying into the system.
2. Buying a map from an outfitter.

It turns out there are no outfitters that will sell you a map to Fah Soom.